c4/docker-compose.yml

```diff
@@ -1,38 +1,22 @@
 version: "3.7"
 volumes:
-  connector_db_data:
-  connector_logs:
-  patientlist_db_data:
-  patientlist_logs:
-  idmanager_logs:
-  store_db_data:
-  store_logs:
-secrets:
-  mainzelliste.docker.conf:
-    file: ../internal-configuration/mainzelliste.conf
-  magicpl.docker.xml:
-    file: ../internal-configuration/magicpl.xml
-  dktk_bridgehead_info.docker.xml:
-    file: ../internal-configuration/bridgehead-common.xml
-  centralSearchPublicKey:
-    file: ../internal-configuration/centralSearchPublicKey.der
-  proxy.docker.xml:
-    file: ../internal-configuration/proxy.xml
+  c4-connector-db-data:
+  c4-connector-logs:
+  patientlist-db-data:
+  patientlist-logs:
+  id-manager-logs:
+  c4-store-db-data:
+  c4-store-logs:
 
 services:
   traefik:
-    container_name: bridgehead_traefik

```

Comment:
Did you forget to commit or change the start and install bridgehead? Currently the updates do not work because they search for a unit with the name bridgehead.